### PR TITLE
It translation 0002

### PR DESCRIPTION
**Pull request by codeadmin-peritiae**
api_docs/index.md translation and typos

### CODE REVIEW

File: site/it/about/bib.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+# White Papers su TensorFlow
+
+Questo documento identifica i white papers su TensorFlow.
```
Comment (bhack) [inline site/it/about/bib.md]: If you want still maintain It literal It Is better `individua`

File: site/it/about/bib.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+## Machine Learning su Larga Scala su Sistemi Distribuiti Eterogenei
+
+[Access this white paper.](https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/45166.pdf)
+
+**Abstract:** TensorFlow è un'interfaccia per descrivere algoritmi 
+ed implementazioni di machine learning ed un'implementazione per
```
Comment (bhack) [inline site/it/about/bib.md]: You are repeating implementazioni here.

File: site/it/about/bib.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
+# White Papers su TensorFlow
+
+Questo documento identifica i white papers su TensorFlow.
+
+## Machine Learning su Larga Scala su Sistemi Distribuiti Eterogenei
+
+[Access this white paper.](https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/45166.pdf)
+
+**Abstract:** TensorFlow è un'interfaccia per descrivere algoritmi
```
Comment (bhack) [inline site/it/about/bib.md]: The original was express so esprimere.

File: site/it/about/bib.md (hunk 1 above)
Comment (bhack) [reply site/it/about/bib.md]: White paper in the non translation list

Comment (bhack) [reply site/it/about/bib.md]: Check also paper(s) and other not translating the terms https://accademiadellacrusca.it/it/consulenza/plurale-dei-forestierismi-non-adattati/8

File: site/it/about/bib.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+**Abstract:** TensorFlow è un'interfaccia per descrivere algoritmi 
+ed implementazioni di machine learning ed un'implementazione per
+eseguire questi algoritmi.
+Una computazione descritta con TensorFlow può essere eseguita
+senza o con minimi cambiamenti su una grande varietà di sistemi
+eterogenei, dai dispositivi mobili come telefoni e tablet
+fino a grandi sistemi distribuiti di centinaia di macchine
+e migliaia di dispositivi di calcolo come le schede GPU.
+Il sistema è flessibile e può essere usato per esprimere una
```
Comment (bhack) [inline site/it/about/bib.md]: Here you are using esprimere. So I suppose you can align with the intital interpretation

File: site/it/about/bib.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
+# White Papers su TensorFlow
+
+<<<<<<< HEAD
```
Comment (lamberta) [inline site/it/about/bib.md]: Need to resolve merge conflict

Comment (codeadmin-peritiae) [reply site/it/about/bib.md]: Fixed

File: site/it/community/contribute/code_style.md [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+    shapes, dtypes, or ranks devono essere specificati nella descrizione. Vedere
+    la documentazione per i dettagli.
+*   Per migliorare l'usabilità, includere un esempio di utilizzo con intput / output
+    dell'operazione nella sezione Example.
+*   Evitare di fare esplicito uso di  `tf.Tensor.eval` o `tf.Session.run`. Per
+    esempio, per scrivere la logica che dipende dal valore del Tensore, usare il 
+    controllo di flusso di TensorFlow. In alternativa, restringere l'operazione in modo che funzioni 
+    solo quando è abilitata la Eager Execution (`tf.executing_eagerly()`).
+
+Esemio:
```
Comment (bhack) [inline site/it/community/contribute/code_style.md]: Esempio

File: site/it/community/contribute/code_style.md [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+*   Le operazioni devono contenere un commento Python estensibile con le dichiarazioni di 
+    Args and Returns che spieghino il tipo e la semantica di ciascun valore. Eventuali
+    shapes, dtypes, or ranks devono essere specificati nella descrizione. Vedere
+    la documentazione per i dettagli.
+*   Per migliorare l'usabilità, includere un esempio di utilizzo con intput / output
+    dell'operazione nella sezione Example.
+*   Evitare di fare esplicito uso di  `tf.Tensor.eval` o `tf.Session.run`. Per
+    esempio, per scrivere la logica che dipende dal valore del Tensore, usare il 
+    controllo di flusso di TensorFlow. In alternativa, restringere l'operazione in modo che funzioni 
+    solo quando è abilitata la Eager Execution (`tf.executing_eagerly()`).
```
Comment (bhack) [inline site/it/community/contribute/code_style.md]: Check again the no traslate list

File: site/it/community/contribute/code_style.md [hunk 8]
```diff
@@ -0,0 +1,166 @@
...
+*   Gli argomenti Tensori devono essere o un singolo tensore o un 'iterabile' di tensori. 
+    Perchè "Tensore o lista di Tensori" è troppo generico. Vedere `assert_proper_iterable`.
+*   Le operazioni che prendono tensori come argomenti, se stanno usando operazioni C++
+    devono chiamare `convert_to_tensor` per convertire input non-tensori in tensori.
+    Notare che nella documentazione gli argoemnti sono ancora descritti come un oggetto `Tensor` di uno
+    specifico dtype.
+*   Ogni operazione Python deve avere un `name_scope`. Come si può vedere sotto, passare il nome
+    dell'operazione come una stringa.
+*   Le operazioni devono contenere un commento Python estensibile con le dichiarazioni di 
+    Args and Returns, che spieghino il tipo e la semantica di ciascun valore. Eventuali
```
Comment (bhack) [inline site/it/community/contribute/code_style.md]: Args and Returns?

Comment (codeadmin-peritiae) [reply site/it/community/contribute/code_style.md]: My approach to translation was to explain in Italian how to produce code fully understandable to every programmer in the world so, written in English.
Therefore, i managed Args and Returns as sections of the suggested coding standard, and I leaved them in English. Do you agree?

Comment (bhack) [reply site/it/community/contribute/code_style.md]: I don't know.
Probably the untraslated Is Args e Returns without (and). 
But here I don't think it Is the reserverd syntax of the programming language but more the concepts of arguments and returns.
So strictly what Is the difference between talking about argomenti and ‘valori di ritorno’ and i.e. return as reserverd word of the programming lang?

Comment (bhack) [reply site/it/community/contribute/code_style.md]: @mcaci What do you think?

Comment (codeadmin-peritiae) [reply site/it/community/contribute/code_style.md]: I agree with you, the sentence "...dichiarazioni di Args e Returns, che spieghino..." into explanation plain text sounds good. --> I'm going to update the document according to that...

To answer your question, in my opinion, we have three scenarios to deal with:
(a) reserved language tokens, i.e. 'return' in C/C++ --> no translation, no rewording, no capital initial
(b) strongly suggested tokens (part of the coding guidelines) that should be used in the code --> no translation, no rewording, capital initial i.e. 'Args' and 'Returns' 
(c) similar english words into plain text explanations, i.e. 'return values' meaning 'the values that are returned' -->  translated and maybe reworded

**Comment by lamberta** [comment]
Thanks for the updates. Can you make separate pull requests for new files? It's too difficult for reviewers if new files are being added during review.

Also, I would concentrate efforts on the notebook tutorials since these are a more interesting (and more viewed) part of the site. Let me know if you need guidance there. I'd start the quickstarts, then Keras, and work your way down the left nav.
Thanks!

**Comment by codeadmin-peritiae** [comment]
Roger. 
I'll make separate pull requests for new files.
If you, please, can consolidate these updates merging this branch, I'll proceed with notebook tutorials starting from the quickstarts where I found some possible improvements.

File: site/it/community/contribute/code_style.md (hunk 8 above)
Comment (mcaci) [reply site/it/community/contribute/code_style.md]: Sorry for the delay, I have seen your comment now @bhack and @codeadmin-peritiae subsequent change and I have nothing to add, I agree with the fact that "Args e Returns" in the full context fits better than "Args and Returns"

**Comment by codeadmin-peritiae** [comment]
Even if I don't see any review from @MarkDaoust, I suppose the work is blocked because of the pull request including more than one file. I'm closing it and re-submit in a different way.

**Comment by lamberta** [comment]
Generally, it's easier for one file per pull request.

Once the community has reviewed and approved, a Googler needs to actually do the merge and we have a whole convoluted process for that that requires a bit of babysitting. Consequently, depending how busy we are, PRs can wait around for a while after the 'lgtm' is applied. Mark and I are added automatically as repo maintainers, but community translations don't necessarily require both our reviews. We try to check the tensorflow/docs pull requests on a regular cadence to merge the ones that are ready.
